playbooks/collect_instance_statistics/collect_appserver_data.yml

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
@pcockwell, why is this duplicating the existing `collect_activity.yml` playbook? Can't the same be used as-is?

Reply:
It isn't duplicating `collect_activity.yml`. See [here](https://github.com/open-craft/opencraft/pull/597/files/3610cfa06163e7ad2f02c2b94f2d15dabbf11f65#diff-2bdd2de6ab95d69edaef3e826f276c35R23) where it passes in an additional parameter to the script. Other than this, yes it is quite similar. I can likely factor out a majority of this playbook, and simply include a sub-playbook for the differences

Reply:
@pcockwell, the difference between this playbook and the existing one is below.

```diff
-- collect_activity/collect_activity.yml       2020-06-30 23:31:46.444239912 +0530
+++ collect_instance_statistics/collect_appserver_data.yml      2020-06-30 23:31:46.444239912 +0530
@@ -11,18 +11,19 @@
     # We have to do a copy+command pair here (instead of script) in order to get
     #   separate channels for stdout and stderr.
     # See also: https://github.com/ansible/ansible-modules-core/pull/4000/files
-    - name: Copy stats script
+    - name: Copy collect_activity script
       copy:
-        src: stats.py
+        src: ../collect_activity/stats.py
         dest: /tmp/collect_activity.py
       become: yes
       become_user: www-data
 
-    - name: Run stats script
+    - name: Run collect_activity script
       command: >
         /edx/bin/python.edxapp /tmp/collect_activity.py
           --config-section {{ inventory_hostname }}
           --out {{ remote_output_filename }}
+          --skip-hits-statistics
       register: stats_results
       become: yes
       become_user: www-data
@@ -36,7 +37,7 @@
     - name: Fetch stats
       fetch:
         src: '{{ remote_output_filename }}'
-        dest: '{{ local_output_dir }}/{{ inventory_hostname }}'
+        dest: '{{ local_output_dir }}/user_statistics'
         flat: yes
       register: fetch_results
```

So is there a reason why we can't use the existing playbook? The only meaningful change I see here is the `--skip-hits-statistics` flag and if it is not possible to strip that out, can't we just leave it as-is and just add the new fields in the final output generated by the `instance_statistics_csv` command?